backend/src/main/java/com/google/models/Alert.java

```diff
@@ -27,14 +27,16 @@
   public static final String ALERT_ENTITY_KIND = "alert";
   public static final String STATUS_PROPERTY = "status";
   public static final String ANOMALIES_LIST_PROPERTY = "anomaliesList";
-  public static final String RESOLVED_STATUS = "resolved";
-  public static final String UNRESOLVED_STATUS = "unresolved";
-  
+  public static enum StatusType {
+    resolved,
+    unresolved
```

Comment:
fyi 
Enums are a type and the enum name should start with a capital. Enum members are constants and their text should be all-uppercase.

i think this is not just in google but pretty common coding convention.

Reply:
okay, i'll fix it! thanks for letting me know.